patches/tModLoader/Terraria/Projectile.cs.patch

```diff
@@ -555,6 +675,24 @@
  				localAI[0] = 1f;
  			}
  		}
+@@ -26715,7 +_,7 @@
+ 					break;
+ 			}
+ 
+-			float meleeSpeed = player.meleeSpeed;
++			float meleeSpeed = 1f / player.meleeSpeed;
```

Comment:
Do we need to worry about player.meleeSpeed ever being zero?
Similar to another review comment, why the change?

Reply:
(Resolved)